lib/network/kmsAWS/Client.ts

```diff
@@ -0,0 +1,256 @@
+'use strict'; // eslint-disable-line
+/* eslint new-cap: "off" */
+
+import errors from '../../errors';
+import { Agent } from 'https';
+import { KMS, AWSError } from 'aws-sdk';
+import * as werelogs from 'werelogs';
+import assert from 'assert';
+
+type TLSVersion = 'TLSv1.3' | 'TLSv1.2' | 'TLSv1.1' | 'TLSv1';
+
+/**
+ * Normalize errors according to arsenal definitions
+ * @param err - an Error instance or a message string
+ * @returns - arsenal error
+ */
+function _arsenalError(err: string | Error) {
```

Comment:
nitpick: Could be worth extracting the prefix as an argument and making this function common for `kmsAWS` and `kmip` to avoid duplication (if there is an obvious place to put it).

Could be used in other places eventually.